pkg/api/mlflow/api/response/metric.go

```diff
@@ -1,7 +1,7 @@
 package response
 
 import (
-	"github.com/G-Research/fasttrackml/pkg/api/mlflow/dao/models"
+	"github.com/G-Research/fasttrackml/pkg/common/dao/models"
```

Comment:
Nice refactor -- I'd say we don't even need "common/dao" in path and could just place under pkg, eg `pkg/models`, `pkg/repositories`, etc

Reply:
+1

Reply:
additionally, we need to move `api/mlflow/dao/repositories` to a common location